src/library-authoring/LibraryLayout.tsx

```diff
@@ -23,12 +23,8 @@ const LibraryLayout = () => {
     throw new Error('Error: route is missing libraryId.');
   }
 
-  const location = useLocation();
   const context = useCallback((childPage) => (
     <LibraryProvider
-      /** We need to pass the pathname as key to the LibraryProvider to force a
-       * re-render when we navigate to a new path or page. */
-      key={location.pathname}
```

Comment:
This fixes the issue where the LibraryProvider was being remounted when the location.pathname changes.

Reply:
Why were we trying to force a re-render before?

Reply:
When we navigated to the Collection Page before, the first render still had `collectionId = undefined`. Remounting the context provider fixed that. It wasn't the best workaround, but it didn't cause many side effects at the time.